core/src/epicli/data/common/ansible/playbooks/roles/rabbitmq/tasks/update-erlang-cookie.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Update erlang cookie
+  template:
+    src: erlang-cookie.j2
+    dest: /var/lib/rabbitmq/.erlang.cookie
+    owner: "{{ specification.rabbitmq_user }}"
+    group: "{{ specification.rabbitmq_group }}"
+    mode: u=r,go=
+  register: cookie
+
+- name: Restart service
+  when: cookie.changed
+  service:
+    name: rabbitmq-server
+    state: restarted
```

Comment:
The same as above - I would use handler and notify here.

Reply:
The same as above.